templates/wizards/files/acn/template.acn

```diff
@@ -0,0 +1,3 @@
+%{JS: Util.fileName('%{TargetPath}')} DEFINITIONS ::= BEGIN
```

Comment:
it would be good to also ensure upper case first letter (if possible)

please validate with asn1scc, but I'm pretty sure it requires upper-case letter at the beginning of the identifier.